.config/guardian/.gdnbaselines

```diff
@@ -0,0 +1,93 @@
+{

```

Comment:
Is this file also autogenerated? If yes, I wonder why it does not have a top header like the other file, warning to not modify it manually. If not, ignore me.

Reply:
It's all autogenerated